frontend/app/locales/de/translations.js

```diff
@@ -126,12 +133,26 @@ export default {
     detail: {
       expense: 'Aufwände',
       charges: 'Aufladungen',
-      date: 'Datum',
       effort: 'Aufwand',
       employee: 'Mitarbeiter',
       description: 'Beschreibung',
       acknowledged: 'Bestätigt',
       amount: 'Anzahl Stunden'
+    },
+    admin: {
+      subscriptions: 'Abonnements',
+      'confirm-subscription': 'Bestellung bestätigen',
+      customer: 'Kunde',
+      project: 'Projekt',
+      projects: 'Projekte',
+      billingType: 'Verrechnungs Art',
+      orderedTime: 'Bestellte Zeit',
+      usedTime: 'Verbrauchte Zeit',
+      timeTotal: 'Insgesamte Zeit',
+      unconfirmedTime: 'Ausstehende Zeit',
+      admin: 'Admin',
+      'confirm-success': 'Bestellung akzeptiert.',
+      'confirm-deny': 'Bestellung abgelehnt.'
```

Comment:
Be consistent with your naming. Either camel case or dasherize but don't mix it please

Reply:
:+1: